nin/backend/package.json

```diff
@@ -9,6 +9,7 @@
     "concat-files": "0.1.0",
     "crc": "3.4.0",
     "express": "4.13.4",
+    "fluent-ffmpeg": "^2.0.1",
```

Comment:
Could you lock the version number to one specific version, so our asses won't get bitten in the future?